src/Chapter2/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.11)
+project(Chapter2)
+include(../../cmake/UnitTestingCourseCommon.cmake)
+
+add_executable(${PROJECT_NAME}_program program.cpp fibonacci.cpp)
+add_executable(${PROJECT_NAME}_test test.cpp fibonacci.cpp)
+target_link_libraries(
+    ${PROJECT_NAME}_test
+    GTest::gtest_main
```

Comment:
The course doesn't use `gtest_main`, so I'll update this.

Reply:
I have just started to see your PR. The course uses `gtest_main` in the next chapters. Please take a look at my new PR (https://github.com/ImperialCollegeLondon/unit_testing_Cpp/pull/14) for test fixtures. We might need this library.

Reply:
Ah cool, ok. Well I guess I'll just update it so that it's in sync with the code you present in the course, when your PRs are merged.